docs/examples/disk_encryption.md

```diff
@@ -165,3 +166,58 @@ no longer has to be maintained in secure state.**
     ```shell
     $ losetup -d /dev/<your_disk>
     ```
+
+### Firmware or configuration updates
+
+When change to PCR values is expected (e.g. during firmware update), one should
+back up and restore the encryption key. As of now, Fobnail Token can be bounded
+to just one platform, and after update machine becomes a new platform as far as
```

Comment:
@krystian-hebel do we have set of information to which we bind? I guess it would be very useful. Also I immediately think about binding policy which for example can attest/provide secret to all my PC Engines devices with one Fobnail Token, is it possible?